target/linux/ath79/dts/qca9533_tplink_tl-wr841hp-v3.dts

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tplink,tl-wr841hp-v3", "qca,qca9533";
+	model = "TP-Link TL-WR841HP v3";
+
+	aliases {
+		label-mac-device = &wmac;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wifi {
+			label = "green:wifi";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		led_power: power {
+			label = "green:power";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+			default-state = "on";
```

Comment:
Why do you set default-state to on?

Reply:
because factory firmware is default-state on also.

Reply:
Please _keep_ the default-state = on

Reply:
Why do we set them off as a default in DTS then?

Reply:
@adschm I think this whole inconsistent default state on/off does not make sense...

Reply:
I do not understand the question. The default for _any_ LED is "off", since it would not make sense to have _all_ LEDs enabled by default. And then we use the default-state keyword to define for which LEDs we would have a different behavior.

Reply:
And please open a separate thread somewhere for this discussion, so we do not confuse submitters with off-topic taste discussions. default-state = on is correct here.